tardis/montecarlo/montecarlo_numba/r_packet.py

```diff
@@ -133,21 +132,21 @@ def calculate_distance_line(
 ):
     """
     Calculate distance until RPacket is in resonance with the next line
+    
     Parameters
     ----------
-    r_packet: RPacket
-    comov_nu: float
+    r_packet : tardis.montecarlo.montecarlo_numba.r_packet.RPacket
+    comov_nu : float
         comoving frequency at the CURRENT position of the RPacket
-    nu_line: float
+    nu_line : float
         line to check the distance to
-    time_explosion: float
+    time_explosion : float
         time since explosion in seconds
-    is_last_line: bool
+    is_last_line : bool
         return MISS_DISTANCE if at the end of the line list
```

Comment:
Move this above - in between comov_nu and nu_line.

We should specify parameters in same order as in function definition.